src/svgcontext.js

```diff
@@ -334,7 +336,7 @@ export class SVGContext {
       height *= -1;
     }
 
-    this.rect(x, y, width - 0.5, height - 0.5, this.attributes);
```

Comment:
I had to take out these adjustments to get Barlines to behave as expected. Were these to mitigate anti-aliasing artifacts?